sg.tf

```diff
@@ -29,3 +29,45 @@ resource "aws_security_group_rule" "self_egress" {
   self              = true
   security_group_id = "${aws_security_group.security_group.id}"
 }
+
+resource "aws_security_group_rule" "cb_ingress" {
+  count                    = "${length(var.sg_cb_ingress_rule)}"
+  type                     = "ingress"
+  cidr_blocks              = ["${split(",",lookup(var.sg_cb_ingress_rule[count.index],"cidr_blocks"))}"]
+  protocol                 = "${lookup(var.sg_cb_ingress_rule[count.index],"protocol")}"
+  from_port                = "${lookup(var.sg_cb_ingress_rule[count.index],"from_port")}"
+  to_port                  = "${lookup(var.sg_cb_ingress_rule[count.index],"to_port")}"
+  security_group_id        = "${aws_security_group.security_group.id}"
+}
+
+resource "aws_security_group_rule" "ssg_ingress" {
```

Comment:
Why are we creating security group rules inside the module? Rules should be created at the level above where the module is being called from.